Library/Formula/protobuf.rb

```diff
@@ -89,19 +103,37 @@ def install
     end
   end
 
-  test do
-    def testdata; <<-EOS.undent
-      package test;
-      message TestCase {
-        required string name = 4;
-      }
-      message Test {
-        repeated TestCase case = 1;
-      }
-      EOS
+  if build.devel?
```

Comment:
I originally tried this inside the "test do" block and it didn't seem to work. Any idea why?